src/Button.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import '@testing-library/jest-dom/extend-expect'
+import { render } from '@testing-library/react'
+import Button from './Button'
+
+describe('Button', () => {
+  it('renders Button correctly', () => {
```

Comment:
You dont't need to write Button, you can just say: `it('renders correctly, ()={...}`
Jest will use the "describe"-part and replace the "it" with it. Have a look at the test results, I hope you can understand what I mean.